### PR TITLE
Add some very basic navigation tests

### PR DESCRIPTION
**Pull request by grittaweisheit**
add very basic tests for the navBar and the home screen floating button.

Please check the branch out and test if the test run on your machine, too. :)

### CODE REVIEW

File: app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt [hunk 1]
```diff
@@ -20,6 +20,7 @@ import androidx.compose.ui.input.pointer.pointerInput
 import androidx.compose.ui.layout.onGloballyPositioned
 import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.dp
+import com.example.mytrainingpal.components.svgs.getBodyMusclesWithPain
```
Comment (MichaelBuessemeyer) [inline app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt:23]: I am pretty sure this import just broke the master yesterday. Please remove it and try whether the build still works. If it still works, please delete this import

Comment (grittaweisheit) [reply app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt:23]: If I remove the import,  getBodyMusclesWithPain(soreMuscles) is not found which prevents the app from building and starting

Comment (MichaelBuessemeyer) [reply app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt]: but there should be no svgs folder :/ 

The file `BodyMuscles.kt` should be at `app/src/main/java/com/example/mytrainingpal/components/BodyMuscles.kt`

Comment (grittaweisheit) [reply app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt]: I found the issue. It was in my local BodyMuscles.kt :)

Comment (grittaweisheit) [reply app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt]: It's package read ...components.svg

Comment (MichaelBuessemeyer) [reply app/src/main/java/com/example/mytrainingpal/components/MusclePainWidget.kt]: yeah was my mistake. I shared the file like this 😓 ....

**Review by MichaelBuessemeyer — APPROVED** [review]
LGTM, lets mergy

**Pull request merged**